noah/src/main/scala/edu/uci/ics/cloudberry/noah/feed/CmdLineAux.java

```diff
@@ -33,4 +35,17 @@ public static BufferedWriter createWriter(String fileName) throws IOException {
                 new OutputStreamWriter(zip, "UTF-8"));
         return bw;
     }
+
+    public static Long[] parseID(String[] stringIDs) throws CmdLineException{
```

Comment:
I found a better way here: http://stackoverflow.com/questions/754294/convert-an-array-of-primitive-longs-into-a-list-of-longs